### PR TITLE
Fix 4952

### PR DESCRIPTION
**Pull request by Limezy**
Fixes : #4952

### CODE REVIEW

**Comment by Limezy** [comment]
image

File: shared/editor/embeds/GoogleLookerStudio.tsx [hunk 1]
```diff
@@ -3,30 +3,30 @@ import Frame from "../components/Frame";
...
       border
     />
   );
 }
 
-GoogleDataStudio.ENABLED = [
+GoogleLookerStudio.ENABLED = [
   new RegExp(
-    "^https?://datastudio\\.google\\.com/(embed|u/0)/reporting/(.*)/page/(.*)(/edit)?$"
+    "^https?://lookerstudio\\.google\\.com/(embed|u/0)/reporting/(.*)/page/(.*)(/edit)?$"
```
Comment (tommoor) [inline shared/editor/embeds/GoogleLookerStudio.tsx:28]: Can we retain support for the old embeds?

Comment (Limezy) [reply shared/editor/embeds/GoogleLookerStudio.tsx]: I have tried something below but am not sure about how they are kept in the database.
Maybe it's more complex than a simple regex OR... And requires a kind of migration ?

Comment (tommoor) [reply shared/editor/embeds/GoogleLookerStudio.tsx]: This is fine 👍

**Review by tommoor — APPROVED** [review]

File: shared/editor/embeds/GoogleLookerStudio.tsx (hunk 1 above)
Comment (Limezy) [reply shared/editor/embeds/GoogleLookerStudio.tsx]: Great !

**Pull request merged**